src/core/positioning/nmeagnssreceiver.cpp

```diff
@@ -85,3 +105,61 @@ void NmeaGnssReceiver::handleStopLogging()
 {
   mLogFile.close();
 }
+
+void NmeaGnssReceiver::handleIMUCorrection( const QString &sentence )
+{
+  static const int PARAMETER_STATUS = 19;
+
+  static const int IMU_KQGEO_STATUS_OK = 1026;
+  static const int IMU_KQGEO_STATUS_OK_NEW = 1967106;
+
+  QStringList parameters = sentence.split( ',' );
+  if ( parameters.size() < PARAMETER_STATUS )
```

Comment:
I think here it should be size() <= PARAMETER_STATUS, otherwise you might end up with a PARAMETER_STATUS (i.e. PARAMETER_INDEX ;) ) that's out of bound.